src/app.module.ts

```
@@ -17,6 +17,7 @@ import { PaymentModule } from './payment/payment.module';
 import { NotificationsModule } from './notifications/notifications.module';
 import { NotificationSettingsModule } from './notification-settings/notification-settings.module';
 import { FreelancerProfileModule } from './freelancer-profile/freelancer-profile.module';
+import { PostService } from './post/post.service';
```

ESLint

[error] 'PostService' is defined but never used.

(@typescript-eslint/no-unused-vars)